tests/conftest.py

```diff
@@ -0,0 +1,33 @@
+import tarfile
+import os
+
+tar_f = os.path.join(os.path.dirname(__file__), 'data',
```

Comment:
Aside: If you haven't tried out pathlib yet, I'd highly recommend it as an alternative to `os.path`.

Reply:
I have; I just always forget about it... old habits die hard ;)